Integrations/AlienVault_USM_Anywhere/AlienVault_USM_Anywhere.py

```diff
@@ -199,6 +204,34 @@ def parse_events(events_data):
     return events
 
 
+def dict_value_to_int(target_dict: Dict, key: str):
+    """
+    :param target_dict: A dictionary which has the key param
+    :param key: The key that we need to convert it's value to integer
+    :return: The integer representation of the key's value in the dict params
+    """
+    try:
+        if target_dict:
+            value = target_dict.get(key)
+            if value:
+                target_dict[key] = int(value)
+                return target_dict[key]
+    except ValueError:
+        raise Exception(f'This value for {key} must be an integer.')
+
+
+def run_function_date_format(function, arg):
+    """
+    :param function: A function that has the arg date_format
+    :param arg: An argument necessary for the function
+    :return: The result of the application expression
+    """
+    try:
```

Comment:
lets convert this to a loop in order to support more future formats without a lot of changes